### PR TITLE
hello world first pull request

### PR DESCRIPTION
**Pull request by zewdu444**
in this pull feature request

-  index html file with heading were added
- style sheet were set for heading
- linter file configured
- used microverse readme template 
- added gitignore file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [x] As stated the project requirement kindly make sure that you added A .css file that includes `one class` with styles for your header.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @zewdu444    , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (zewdu444) [reply index.html]: Thank you, @Abenezer-Tilahun , for your rapid and thorough review comment regarding the CSS file class and what I missed on the header. I have added class to the header file.

Comment (Abenezer-Tilahun) [reply index.html]: Hello @zewdu444 now you submit for re-review and it will be approved.

Comment (zewdu444) [reply index.html]: @Abenezer-Tilahun, I'm sorry to inform you that I was unable to locate a re-review request option on the student dashboard. Could you perhaps direct me?

**Pull request closed without merging**